Services/CO.CDP.Person.WebApi/Api/Person.cs

```diff
@@ -54,6 +63,16 @@ await useCase.Execute(personId)
         app.MapPut("/persons/{personId}",
                 (Guid personId, UpdatePerson updatedPerson) =>
                 {
+                    if (!_persons.ContainsKey(personId))
+                    {
+                        return Results.NotFound();
+                    }
+
+                    if (updatedPerson == null)
```

Comment:
updatedPerson will never be null from ASP.NET pipeline.
Also, BadRequest check should be before 404